src/Engine/GameAction.java

```diff
@@ -675,5 +745,11 @@ public ActionType getType()
       // action with an actor, target location, etc.
       return GameAction.ActionType.OTHER;
     }
+
+    @Override
+    public String toString()
+    {
+      return "[Perform CO Ability]";
```

Comment:
Wouldn't hurt to retain the ability so you can print the name here

Reply:
Okay